hazelcast/src/test/java/com/hazelcast/map/InterceptorTest.java

```diff
@@ -341,6 +341,7 @@ public Iterable<Integer> loadAllKeys() {
     }
 
     static class EntryPutProcessor implements EntryProcessor<Integer, String, String> {
+        private static final long serialVersionUID = 1L;
```

Comment:
this seems like an unrelated change to me.

Reply:
c2772b2416f0412b1a05a0d212ec295bcafe06d9